docs/Configuration.md

```diff
@@ -1202,10 +1202,10 @@ Each plugin is an object or a function that returns an object with the following
 - `configResolved`: React to the resolved configuration
 - `configureJest`: Configure Jest with access to both project and global config
 - `transform`: Transform code before it's executed
-- `apply`: Hook into watch mode lifecycle events (replaces watch plugin `apply`)
-- `getUsageInfo`: Define interactive watch mode key bindings (replaces watch plugin `getUsageInfo`)
-- `run`: Execute custom actions in watch mode (replaces watch plugin `run`)
-- `onKey`: Handle key presses in watch mode (replaces watch plugin `onKey`)
+- `subscribeToWatchMode`: Subscribe to watch mode lifecycle events (migrated from watch plugin `apply`)
+- `getUsageInfo`: Define interactive watch mode key bindings (same as watch plugin `getUsageInfo`)
+- `executeInteractiveAction`: Execute custom actions in watch mode (migrated from watch plugin `run`)
+- `onKey`: Handle key presses in watch mode (same as watch plugin `onKey`)
```

Comment:
There is no onKey in watch plugin, see https://jestjs.io/docs/watch-plugins

Reply:
Seems like documentation  https://jestjs.io/docs/watch-plugins misses onKey method, can you please adjust this doc too